packages/sit-onyx/src/components/OnyxNavBar/modules/OnyxUserMenu/types.ts

```diff
@@ -11,4 +11,9 @@ export type OnyxUserMenuProps = {
    * Optional user description that is displayed when the menu is opened.
    */
   description?: string;
+  /**
+   * If the flyout is expanded or not.
+   * If `undefined`, the state will be managed internally.
+   */
+  open?: boolean;
```

Comment:
I don't see this property used anywhere?

Reply:
Will do in follow-up